eox_hooks/models.py

```diff
@@ -0,0 +1,34 @@
+"""Models module for eox-hooks."""
+import uuid
+
+from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
+
+from eox_hooks.constants import Status
+
+
+@python_2_unicode_compatible
+class HookExecutionAudit(models.Model):
+    """Model used to audit the eox-hooks actions."""
+
+    key = models.UUIDField(
+        unique=True,
+        editable=False,
+        default=uuid.uuid4,
+        verbose_name="Public identifier",
+    )
+
+    status = models.PositiveIntegerField(
+        choices=Status.choices(), default=Status.SUCCESS, editable=False,
+    )
+    executed_at = models.DateTimeField(auto_now_add=True, editable=False)
+    hook_name = models.CharField(max_length=150)
+    task_name = models.CharField(max_length=150)
+    tenant_domain = models.CharField(max_length=150)
+    task_duration = models.FloatField(null=True)
+
+    objects = models.Manager()
```

Comment:
this is the default value , isn't it ?

Reply:
Yes! but pylint throws an error when using objects.create.
E: 19,22: Class 'HookExecutionAudit' has no 'objects' member (no-member)
I fixed it adding the default

Reply:
Try to suppress the warning instead of add this